src/marshaller/HipieDDL.js

```diff
@@ -1320,7 +1320,7 @@
             if (titleWidget) {
                 var title = titleWidget.title();
                 var titleParts = title.split(" (");
-                if (titleParts[0] === "" && titleParts.length > 1) {
+                if (typeof titleWidget.useParamFlyoutTitles === "function" && titleWidget.useParamFlyoutTitles()) {
```

Comment:
This appears to be a breaking change when the function doesn't exist or is false?